hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/evm/Evm46ValidationSuite.java

```diff
@@ -1474,4 +1475,92 @@ final Stream<DynamicTest> directCallToSystemAccountResultsInSuccessfulNoOp() {
                                 .status(SUCCESS)
                                 .contractCallResult(resultWith().gasUsed(GAS_LIMIT_FOR_CALL))));
     }
+
+    @HapiTest
+    final Stream<DynamicTest> testCallOperationsForSystemAccounts() {
+        final var contract = "CallOperationsCheckerSuccess";
+        final var functionName = "call";
+        final HapiSpecOperation[] opsArray = new HapiSpecOperation[callOperationsSuccessSystemAccounts.size()];
+        for (int i = 0; i < callOperationsSuccessSystemAccounts.size(); i++) {
+            int finalI = i;
+            opsArray[i] = withOpContext((spec, opLog) -> allRunFor(
+                    spec,
+                    contractCall(
+                                    contract,
+                                    functionName,
+                                    mirrorAddrWith(callOperationsSuccessSystemAccounts.get(finalI)))
+                            .hasKnownStatus(SUCCESS)
+                            .via("callTest" + finalI)));
+        }
```

Comment:
Maybe we could extract the common logic into a separate method?

Also do we need the ".via("callTest" + finalI)));"  part? Seems like we are not doing any verifications with it.

Reply:
Good suggestion. I extracted the common looping logic in a new method. And also removed the .via as we don't need it.